2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Configure stats pid to 6305
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Loading settings from /Users/volodimir/.config/wandb/settings
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Loading settings from /Users/volodimir/Desktop/Bachelor/wandb/settings
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-19 16:20:45,940 INFO    MainThread:6305 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'training.py', 'program_abspath': '/Users/volodimir/Desktop/Bachelor/training.py', 'program': '/Users/volodimir/Desktop/Bachelor/training.py'}
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:_log_setup():526] Logging user logs to /Users/volodimir/Desktop/Bachelor/wandb/run-20240219_162045-j09wp5pm/logs/debug.log
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/volodimir/Desktop/Bachelor/wandb/run-20240219_162045-j09wp5pm/logs/debug-internal.log
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:init():566] calling init triggers
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'architecture': 'resnet18', 'epochs': 40, 'batch_size': 32, 'dataset': '29DVOHC6Q45Q', 'pretrained_model': True}
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:init():616] starting backend
2024-02-19 16:20:45,941 INFO    MainThread:6305 [wandb_init.py:init():620] setting up manager
2024-02-19 16:20:45,944 INFO    MainThread:6305 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-19 16:20:45,947 INFO    MainThread:6305 [wandb_init.py:init():628] backend started and connected
2024-02-19 16:20:45,954 INFO    MainThread:6305 [wandb_init.py:init():720] updated telemetry
2024-02-19 16:20:45,972 INFO    MainThread:6305 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 16:20:46,431 INFO    MainThread:6305 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 16:20:46,547 INFO    MainThread:6305 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 16:20:46,547 INFO    MainThread:6305 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 16:20:49,593 INFO    MainThread:6305 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 16:20:49,593 INFO    MainThread:6305 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 16:20:49,593 INFO    MainThread:6305 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 16:20:49,594 INFO    MainThread:6305 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 16:20:49,594 INFO    MainThread:6305 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 16:58:01,895 INFO    MainThread:6305 [wandb_run.py:_config_callback():1343] config_cb ('_wandb', 'visualize', 'confusion_matrix') {'panel_type': 'Vega2', 'panel_config': {'panelDefId': 'wandb/confusion_matrix/v1', 'fieldSettings': {'Actual': 'Actual', 'Predicted': 'Predicted', 'nPredictions': 'nPredictions'}, 'stringSettings': {'title': ''}, 'transform': {'name': 'tableWithLeafColNames'}, 'userQuery': {'queryFields': [{'name': 'runSets', 'args': [{'name': 'runSets', 'value': '${runSets}'}], 'fields': [{'name': 'id', 'fields': []}, {'name': 'name', 'fields': []}, {'name': '_defaultColorIndex', 'fields': []}, {'name': 'summaryTable', 'args': [{'name': 'tableKey', 'value': 'confusion_matrix_table'}], 'fields': []}]}]}}} None
2024-02-19 16:58:07,817 WARNING MsgRouterThr:6305 [router.py:message_loop():77] message_loop has been closed
